libs/model/src/policies/handlers/notifyAddressOwnershipTransferred.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  EventHandler,
+  notificationsProvider,
+  WorkflowKeys,
+} from '@hicommonwealth/core';
+import { ZodBoolean } from 'zod';
+import { models } from '../../database';
+
+export const notifyAddressOwnershipTransferred: EventHandler<
+  'AddressOwnershipTransferred',
+  ZodBoolean
+> = async ({ payload }) => {
+  // TODO: should we ignore when old_user_email is null?
+  if (!payload.old_user_email) return true;
+
+  const community = await models.Community.findOne({
+    where: { id: payload.community_id },
+  });
+  if (!community) return false;
+
+  const provider = notificationsProvider();
+  const res = await provider.triggerWorkflow({
+    key: WorkflowKeys.AddressOwnershipTransferred,
```

Comment:
I don't see a workflow on Knock for this.

Reply:
I'll add an entry to the spreadsheet notifications v2

Reply:
Same as for contests, if this event is only used for notifications we should not emit it until the workflow is complete.

Reply:
added to the ignore list